grain_size_tools/GrainSizeTools_script.py

```diff
@@ -30,9 +30,9 @@
 #                                                                              #
 # ============================================================================ #
 
-import plots as plots
-import tools as tools
-from piezometers import quartz, olivine, calcite, feldspar
+from . import plots
+from . import tools
```

Comment:
Here, we want to import plot and tools from the grainsize tools package, rather than from a global package called `plots` or `tools` (e.g. if someone created a package called `tools` and installed it, there would be name clashes). Check out: https://docs.python.org/3/tutorial/modules.html#intra-package-references